pythonosc/dispatcher.py

```diff
@@ -45,7 +45,12 @@ def handlers_for_address(self, address_pattern):
     pattern = pattern + '$'
     pattern = re.compile(pattern)
     matched = [
-        handler for addr, handler in self._map.items() if pattern.match(addr)]
+        handler for addr, handler in self._map.items()
+        if ( pattern.match(addr) # wildcards could be in the incoming pattern
+             # .. or the mapped pattern
+             or (('*' in addr) and re.match(addr.replace('*','[^/]*?/*'), address_pattern))
```

Comment:
no worries, python caches the regexps anyway so let's deal with precompiling if the needs come :)